cmd/kube-controller-manager/app/plugins.go

```diff
@@ -43,7 +43,22 @@ func ProbeRecyclableVolumePlugins() []volume.VolumePlugin {
 	// The list of plugins to probe is decided by the kubelet binary, not
 	// by dynamic linking or other "magic".  Plugins will be analyzed and
 	// initialized later.
-	allPlugins = append(allPlugins, host_path.ProbeVolumePlugins()...)
-	allPlugins = append(allPlugins, nfs.ProbeVolumePlugins()...)
+	allPlugins = append(allPlugins, host_path.ProbeVolumePlugins(defaultRecycler(60))...)
```

Comment:
Agree to all three.  This was hastily thrown together to avoid the hard-coded busybox requirement.

Is it worthwhile to override how something gets scrubbed?  I can make a custom kube image for the scrubber.  Do we need more than that?

It might be we just need to override image name.  I can figure out X seconds timeout per gig to scrub in a volume, which removes these magic numbers.